scripts/20210517_validation-object/script.js

```diff
@@ -0,0 +1,394 @@
+/* eslint-disable */
+// Scripts for #408
+
+/**
+ * Part 1: Verify that the database is internally consistent today
+ * for all short text, long text and number fields.
+ * Expect 0 records for all scripts
+ */
+
+// a. customVal === customMin when selectedValidation === 'Minimum'
+// Expect 0 records
+
+db.getCollection('forms').aggregate([
+  {
+    $match: {
+      'form_fields': {$elemMatch: {'fieldType': { $in: ['textfield', 'textarea', 'number'] }}},
```

Comment:
I actually don't think you need `$elemMatch` here. from the [docs](https://docs.mongodb.com/manual/reference/operator/query/elemMatch/#std-label-elemmatch-single-query-condition): 
> If you specify a single query predicate in the $elemMatch expression, and are not using the $not or $ne operators inside of $elemMatch, $elemMatch can be omitted.